the-collab-lab/tcl-73-smart-shopping-list
# Style input fields with Radix

## Description

Style Input Fields using Radix.

## Updates

### Before
Mobile Before:
![Screenshot 2024-04-02 at 5 29 01 PM](https://github.com/the-collab-lab/tcl-73-smart-shopping-list/assets/111473039/aef6cefb-cf24-4937-806e-b2557a9e7f88)

Desktop Before:
![Screenshot 2024-04-02 at 5 29 41 PM](https://github.com/the-collab-lab/tcl-73-smart-shopping-list/assets/111473039/abce9d31-203f-4970-bfd0-93d1a11c7a56)

![Screenshot 2024-04-02 at 5 30 27 PM](https://github.com/the-collab-lab/tcl-73-smart-shopping-list/assets/111473039/831e839e-90ca-4e86-857c-3bfd6348bcba)

![Screenshot 2024-04-02 at 5 30 59 PM](https://github.com/the-collab-lab/tcl-73-smart-shopping-list/assets/111473039/67993ba3-3b62-4adc-862b-8b6f598894d0)

![Screenshot 2024-04-02 at 5 31 25 PM](https://github.com/the-collab-lab/tcl-73-smart-shopping-list/assets/111473039/2a82985a-2ca4-478f-a8f6-e62874fff572)

### After

Mobile After:
![Screenshot 2024-04-02 at 5 21 39 PM](https://github.com/the-collab-lab/tcl-73-smart-shopping-list/assets/111473039/3c7fd5e4-719d-4eae-9434-fb45ed8eaea8)

Desktop After
![Screenshot 2024-04-02 at 5 23 35 PM](https://github.com/the-collab-lab/tcl-73-smart-shopping-list/assets/111473039/46d63074-063b-42fb-a5af-dbe78e1b58df)

![Screenshot 2024-04-02 at 5 24 07 PM](https://github.com/the-collab-lab/tcl-73-smart-shopping-list/assets/111473039/a09cd419-c22b-4ef5-a20b-d88e64e38267)

![Screenshot 2024-04-02 at 5 24 34 PM](https://github.com/the-collab-lab/tcl-73-smart-shopping-list/assets/111473039/7340ed61-3afc-4c66-92b3-37ceb0ffabeb)

![Screenshot 2024-04-02 at 5 24 53 PM](https://github.com/the-collab-lab/tcl-73-smart-shopping-list/assets/111473039/e3d1c173-582b-4fb6-a8af-75b5555c7f5f)


## Testing Steps / QA Criteria

- git pull
- git checkout rs-style-input-fields
- npm ci
- npm start
- On the `Home` page, add a new list and ensure the list gets added
- On the `List` page, search for an item in a list with items in it. *Note: the dark blue on the button that comes up after you start typing will be more of an accessible color once the light/dark mode issue is merged
- On the `Manage List` page, add an new item, and make sure the item is created successfully
- On the `Manage List` page, add a users email, and make sure the list is shared successfully
